.gitignore

```diff
@@ -1,9 +1,17 @@
+# environment
 .DS_Store
 .idea
 .env/
 .env-*/
 env/
 env-*/
+.Hasal
+
+#local file
+agent_status/
+resource/
+upload/
+test.suite
```

Comment:
@ypwalter test.suite is a local file for keeping the test suites which DuT is running. In case there will be diffs blocks code update. It's better to ignore test.suite. If we need to update default test.suite, it's fine to force update.